woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -2980,32 +2980,49 @@ def test_nan_index_error(sample_df_pandas):
 
 
 def test_validate_logical_types(sample_df):
-    df = sample_df[["email", "age"]]
+    df = sample_df[["email", "url", "age"]]
```

Comment:
I think going forward we may want to change our approach here slightly. As we add validation to more logical types, this test is going to get messy. I'm fine with leaving this test as it is for now since it tests that we get data back for multiple series as we expect.

In the future, instead of continuing to modify this test, we might want to add a simpler test just to verify that the proper `LogicalType.validate()` functions get called when we call `validate_logical_types` from the table accessor, without checking the actual values we get back. We can test that the validate functions have the proper behavior with the targeted tests for each specific logical type.

Reply:
sure I don't mind updating the test

Reply:
[0f5cb30](https://github.com/alteryx/woodwork/pull/1285/commits/0f5cb30d881e51fe8364b2995cbb020d43bf1b7d)